geomesa-web/geomesa-web-core/src/main/scala/org/locationtech/geomesa/web/core/SpringScalatraBootstrap.scala

```diff
@@ -30,9 +30,12 @@ trait GeoMesaScalatraServlet extends ScalatraServlet with LazyLogging {
 
   // This may be causing issues within scalatra, to paraphrase a comment: "Wrapped requests are probably wrapped for a reason."
   // https://geomesa.atlassian.net/browse/GEOMESA-1062
-  override def handle(req: HttpServletRequest, res: HttpServletResponse): Unit = req match {
-    case r: HttpServletRequestWrapper => super.handle(r.getRequest.asInstanceOf[HttpServletRequest], res)
-    case _ => super.handle(req, res)
+  override def handle(req: HttpServletRequest, res: HttpServletResponse): Unit = {
+    val wrapped = req match {
+      case r: HttpServletRequestWrapper => new PathHandlingServletRequest(r)
+      case _ => req
+    }
+    super.handle(wrapped, res)
```

Comment:
Can you make this same change in GeoMesaStatsEndpoint.scala?

This code is copied around since the override was stomped by the other Scalatra mixins.